examples/kubernetes/dynamic_provisioning_withdeploymenttype/README.md

```diff
@@ -0,0 +1,80 @@
+## Dynamic Provisioning with DeploymentType
+This example shows how to create a FSx for Lustre filesystem using persistence volume claim (PVC) with DeploymentType specified
+
+### Edit [StorageClass](./specs/storageclass.yaml)
+```
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: fsx-sc
+provisioner: fsx.csi.aws.com
+parameters:
+  subnetId: subnet-0cd1914f0b9041ebe
+  securityGroupIds: sg-07662e27978b3b360,sg-057948b58dd38e68b
+  s3ImportPath: s3://evanzempty
+  s3ExportPath: s3://evanzempty
+  deploymentType: SCRATCH_2
+mountOptions:
+  - flock,noatime
+```
+* subnetId - the subnet ID that the FSx for Lustre filesystem should be created inside.
+* securityGroupIds - a comman separated list of security group IDs that should be attached to the filesystem
+* s3ImportPath(Optional) - S3 data repository you want to copy from S3 to persistent volume
+* s3ExportPath(Optional) - S3 data repository you want to export new or modified files from persistent volume to S3
+* deploymentType: FSx for lustre now provides three deployment types which uses different EC2 instances types
```

Comment:
Add the list of three valid deployment types here?

Reply:
Sure.